Peer vuex-router-sync@2.1.0 wants vuex@>= 0.6.2 < 3

```
$ npm install --save vuex-router-sync
npm WARN peerDependencies The peer dependency vuex@>= 0.6.2 < 3 included from vuex-router-sync will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm ERR! Linux 4.4.0-36-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install" "--save" "vuex-router-sync"
npm ERR! node v4.5.0
npm ERR! npm  v2.15.9
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package vuex@1.0.0-rc.2 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer vuex-router-sync@2.1.0 wants vuex@>= 0.6.2 < 3

npm ERR! Please include the following file with any support request:
npm ERR!     /home/ferri/Developments/domains/d2/npm-debug.log
```

Hi i've got error like that when try to run `npm install --save vuex-router-synx`. Any idea? this is my package.json

```json
"dependencies": {
    "babel-runtime": "^6.0.0",
    "vue": "^1.0.21",
    "vue-router": "^0.7.13",
    "vuex": "^1.0.0-rc.2"
  },
```

Thanks

--- a/package.json
+++ b/package.json
@@ -19,7 +19,7 @@
   },
   "homepage": "https://github.com/vuejs/vuex-router-sync#readme",
   "peerDependencies": {
-    "vuex": ">= 0.6.2 < 3",
+    "vuex": ">= 0.6.2 < 3 || ^1.0.0-rc ||^2.0.0-rc",
     "vue-router": ">=0.7.11"
   }
 }

